### PR TITLE
Fix some universe name printing

### PR DESCRIPTION
**Pull request by SkySkimmer**
Show Universes did not use names in the values of the opt_subst

Overlays:
- https://github.com/LPCIC/coq-elpi/pull/431
- https://github.com/coq-community/paramcoq/pull/106

### CODE REVIEW

**Comment by Alizter** [comment]
Is this just `Univ.Level.pr` -> `Univ.Level.raw_pr` everywhere then?

File: engine/termops.ml [hunk 1]
```diff
@@ -244,7 +244,7 @@ let pr_evar_universe_context ctx =
      str"ALGEBRAIC UNIVERSES:"++brk(0,1)++
      h (Univ.Level.Set.pr prl (UState.algebraics ctx)) ++ fnl() ++
      str"UNDEFINED UNIVERSES:"++brk(0,1)++
-     h (UState.pr_universe_opt_subst (UState.subst ctx)) ++ fnl() ++
+     h (UState.pr_universe_opt_subst prl (UState.subst ctx)) ++ fnl() ++
```
Comment (SkySkimmer) [inline engine/termops.ml:247]: Here is a non renaming change

**Review by Alizter — APPROVED** [review]

**Comment by SkySkimmer** [comment]
@Alizter ping

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**